packages/core/src/Stores/notification-store.js

```diff
@@ -191,7 +191,7 @@ export default class NotificationStore extends BaseStore {
             }
         } else if (has_mt5_account_with_rejected_poa) {
             this.addNotificationMessage(this.client_notifications.poa_rejected_for_mt5);
-        } else if (!['none', 'pending'].includes(document.status)) {
+        } else if (document.status === 'failed') {
```

Comment:
Help to create a tech debt card to move all these hard coded statuses to a constant file and access them from the file.